svelte-case-study.md

```diff
@@ -0,0 +1,203 @@
+
+## Features
+
+The key features of Svelte are:
+
+- No virtual DOM
+  - Omitting a virtual DOM results in faster performance compared to other popular frameworks
+- Reactive Declarative Syntax
+  - Svelte uses a reactive and declarative syntax for building UI components, which makes it easy to understand and use for developers of all skill levels.
+- Small bundle size
+  - This makes it an ideal choice for lightweight applications or for use on slow networks.
+- Built-in animations and  transitions
+  - Includes built-in support for animations and transitions, allowing developers to easily add animations to their applications without requiring additional libraries or plugins.
+- Server-side Rendering
+  - Supports server-side rendering, making it easy to optimize the performance of applications for search engines and users with slow networks.
+- Easy to learn
+  - Svelte has a simple and intuitive syntax, making it easy to learn and use, even for developers new to web development.
+
+When a component is added to the DOM, Svelte sets up the reactive values for that component and starts observing changes to them. When the component is removed from the DOM, Svelte stops observing those values and cleans up any associated resources, freeing up more resources compared to other frameworks.
+
+Svelte uses a system of reactive statements, such as {#if} blocks, to conditionally render or destroy a component based on its reactive values. For example, if a component has a reactive value visible that determines whether it should be displayed or not, you could use an {#if} block to conditionally render the component, like this:
+
+```js
+{#if visible}
+  <HiddenComponent msg="Hello World!"/>
+{/if}
+```
+
+When the value of visible changes, Svelte automatically updates the DOM to reflect the change and render or destroy the component as needed. This allows for easy control over component loading and unloading, and ensures that resources are only used when they are actually required, instead of constantly rendering every component like in some high-budget slideshow.
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/30121693/217382445-d5addc54-21a2-48bd-b613-dd9cac2e657b.png" alt="Svelte's key features"/>
+</p>
```

Comment:
```suggestion
```

Where possible we try to use as little HTML and handle formatting issues with CSS using consistent rules